util/libcrypto.num

```diff
@@ -4193,3 +4193,7 @@ EVP_PKEY_get1_tls_encodedpoint          4139	1_1_0	EXIST::FUNCTION:
 CT_POLICY_EVAL_CTX_set1_cert            4140	1_1_0	EXIST::FUNCTION:CT
 CT_POLICY_EVAL_CTX_set1_issuer          4141	1_1_0	EXIST::FUNCTION:CT
 CT_POLICY_EVAL_CTX_set_shared_CTLOG_STORE 4142	1_1_0	EXIST::FUNCTION:CT
+ECPKPARAMETERS_free                     4143	1_1_0	EXIST::FUNCTION:EC
+ECPARAMETERS_free                       4144	1_1_0	EXIST::FUNCTION:EC
+ECPKPARAMETERS_new                      4145	1_1_0	EXIST::FUNCTION:EC
+ECPARAMETERS_new                        4146	1_1_0	EXIST::FUNCTION:EC
```

Comment:
Really why not? Perhaps we could get away with a generic page for all the ASN1 functions (if so I agree that would be out of scope for this PR)